setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/CommonFiles.swr

```diff
@@ -36,13 +36,15 @@ folder "InstallDir:MSBuild\Microsoft\VisualStudio\Managed"
   file source="$(VisualStudioXamlRulesDir)DotNetCliToolReference.xaml"
   file source="$(VisualStudioXamlRulesDir)EmbeddedResource.BrowseObject.xaml"
   file source="$(VisualStudioXamlRulesDir)EmbeddedResource.xaml"
+  file source="$(VisualStudioXamlRulesDir)CollectedFrameworkReference.xaml"
   file source="$(VisualStudioXamlRulesDir)Folder.xaml"
   file source="$(VisualStudioXamlRulesDir)GeneralBrowseObject.xaml"
   file source="$(VisualStudioXamlRulesDir)GeneralConfiguredBrowseObject.xaml"
   file source="$(VisualStudioXamlRulesDir)LanguageService.xaml"
   file source="$(VisualStudioXamlRulesDir)None.xaml"
   file source="$(VisualStudioXamlRulesDir)None.BrowseObject.xaml"
   file source="$(VisualStudioXamlRulesDir)NuGetRestore.xaml"
+  file source="$(VisualStudioXamlRulesDir)CollectedPackageDownload.xaml"
```

Comment:
Nit: With the new prefix these are no longer sorted correctly

Reply:
Yeah I noticed this when I renamed the others in a pending PR; I'll auto-gen this file or write something that will sort it in that PR.